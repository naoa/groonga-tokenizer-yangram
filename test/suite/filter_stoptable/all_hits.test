register tokenizers/yangram
yangram_register --filter_stoptable stopwords

table_create stopwords TABLE_HASH_KEY ShortText

load --table stopwords
[
{"_key": "a"},
{"_key": "pen"},
{"_key": "this"},
{"_key": "is"}
]

tokenize TokenYaBigramFilterStoptable "This is a pen" NormalizerAuto
