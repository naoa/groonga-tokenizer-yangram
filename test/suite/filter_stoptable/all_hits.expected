register tokenizers/yangram
[[0,0.0,0.0],true]
yangram_register --filter_stoptable stopwords
[[0,0.0,0.0],"TokenYaBigramFilterStoptable"]
table_create stopwords TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table stopwords
[
{"_key": "a"},
{"_key": "pen"},
{"_key": "this"},
{"_key": "is"}
]
[[0,0.0,0.0],4]
tokenize TokenYaBigramFilterStoptable "This is a pen" NormalizerAuto
[[0,0.0,0.0],[]]
